contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -211,6 +193,24 @@ func (t *T) Reset() {
 	t.Spec = Spec{}
 }
 
+// SlaveLabels converts slave attributes into string key/value labels
+func SlaveLabels(attrs []*mesos.Attribute) map[string]string {
+	l := map[string]string{}
+	for _, a := range attrs {
+		if a == nil {
+			continue
+		}
+
+		switch a.GetType() {
```

Comment:
any value in trying to support SET types? they're just comma delimited lists, and k8s has some support for set-based field selectors. not sure how frequently they're applied to nodes that have label values containing sets.